source/OutfitterPanel.cpp

```diff
@@ -332,427 +364,588 @@ double OutfitterPanel::DrawDetails(const Point &center)
+						+ Format::MassString(outfitSpace) + " free.");
+
+				double weaponNeeded = -selectedOutfit->Get("weapon capacity");
+				double weaponSpace = playerShip->Attributes().Get("weapon capacity");
+				if(weaponNeeded > weaponSpace)
+					errors.push_back("Only part of your ship's outfit capacity is usable for weapons. "
+						"You cannot install this outfit, because it takes up "
+						+ Format::CargoString(weaponNeeded, "weapon space") + ", and this ship has "
+						+ Format::MassString(weaponSpace) + " free.");
+
+				double engineNeeded = -selectedOutfit->Get("engine capacity");
+				double engineSpace = playerShip->Attributes().Get("engine capacity");
+				if(engineNeeded > engineSpace)
+					errors.push_back("Only part of your ship's outfit capacity is usable for engines. "
+						"You cannot install this outfit, because it takes up "
+						+ Format::CargoString(engineNeeded, "engine space") + ", and this ship has "
+						+ Format::MassString(engineSpace) + " free.");
+
+				if(selectedOutfit->Category() == "Ammunition")
+					errors.emplace_back(!playerShip->OutfitCount(selectedOutfit) ?
+						"This outfit is ammunition for a weapon. "
+						"You cannot install it without first installing the appropriate weapon."
+						: "You already have the maximum amount of ammunition for this weapon. "
+						"If you want to install more ammunition, you must first install another of these weapons.");
+
+				int mountsNeeded = -selectedOutfit->Get("turret mounts");
+				int mountsFree = playerShip->Attributes().Get("turret mounts");
+				if(mountsNeeded > mountsFree)
+					errors.emplace_back("This weapon is designed to be installed on a turret mount, "
+						"but your ship does not have any unused turret mounts available.");
+
+				int gunsNeeded = -selectedOutfit->Get("gun ports");
+				int gunsFree = playerShip->Attributes().Get("gun ports");
+				if(gunsNeeded > gunsFree)
+					errors.emplace_back("This weapon is designed to be installed in a gun port, "
+						"but your ship does not have any unused gun ports available.");
+
+				if(selectedOutfit->Get("installable") < 0.)
+					errors.emplace_back("This item is not an outfit that can be installed in a ship.");
+
+				// Handle other attributes more generically, if none of the above are the problem.
+				if(errors.empty())
+					for(const pair<const char *, double> &it : selectedOutfit->Attributes())
+					{
+						// If playerShip has fewer of this attribute available than required by the selectedOutfit, add
+						// this attribute to the list of deficiencies.
+						double shipAvailable = playerShip->Attributes().Get(it.first);
+						double outfitRequires = -it.second;
+						if(shipAvailable < outfitRequires)
+							errors.push_back("You cannot install this outfit, because it requires "
+								+ Format::Number(outfitRequires) + " '"
+								+ Format::SimplePluralization(outfitRequires, it.first) + "', and this ship has "
+								+ Format::Number(shipAvailable) + " free.");
```

Comment:
```suggestion
							errors.push_back("You cannot install this outfit, because it requires "
								+ Format::SimplePluralization(outfitRequires, "'" + it.first + "'") + ", and this ship has "
								+ Format::Number(shipAvailable) + " free.");
```
SimplePluralisation adds the number already, creating this issue:

<img width="1280" height="800" alt="Image" src="https://github.com/user-attachments/assets/39d3b4e7-8b6c-4266-b584-493fd12926f3" />

The name should be quoted then passed to the function, no need to add more numbers.

There may be more instances of this issue, I haven't checked..